services/QuillLMS/app/services/classlink_integration/temp_users_importer.rb

```diff
@@ -28,9 +28,10 @@ def run
 
         classlink_instance_id = ClasslinkInstance.find_by(oneroster_application_id: district_hash['oneroster_application_id']).id
 
-        existing_provider_users = ClasslinkClassroom
-          .where(provider_instance_id: classlink_instance_id)
-          .includes(:classroom)
+        existing_provider_users =
```

Comment:
This was a potential bug (would have caused spurious `ActiveRecord::Validation` errors emitted in Sentry); fixing it in case we want to use this script again before finishing QA